config/software/ncurses.rb

```diff
@@ -69,9 +69,8 @@
     patch source: "patch-aix-configure", plevel: 0, env: patch_env
   end
 
-  if mac_os_x? ||
-    # Clang became the default compiler in FreeBSD 10+
-    (freebsd? && ohai['os_version'].to_i >= 1000024)
+  # Clang became the default compiler in FreeBSD 10+
+  if mac_os_x? || (freebsd? && ohai['os_version'].to_i >= 1000024)
```

Comment:
@stevendanna That's a great idea. I would make that change in chef-sugar (or at least propose the helper in that repo).